# This is a wrong kind of PPC, we don ºt want it.

--- vpx_ports/vpx_ports.mk.orig	2021-10-07 01:41:19.000000000 +0800
+++ vpx_ports/vpx_ports.mk	2022-04-24 17:38:34.000000000 +0800
@@ -43,8 +43,8 @@
 PORTS_SRCS-$(VPX_ARCH_ARM) += arm_cpudetect.h
 PORTS_SRCS-$(VPX_ARCH_ARM) += arm.h
 
-PORTS_SRCS-$(VPX_ARCH_PPC) += ppc_cpudetect.c
-PORTS_SRCS-$(VPX_ARCH_PPC) += ppc.h
+PORTS_SRCS-$(VPX_ARCH_PPC64LE) += ppc_cpudetect.c
+PORTS_SRCS-$(VPX_ARCH_PPC64LE) += ppc.h
 
 PORTS_SRCS-$(VPX_ARCH_MIPS) += mips_cpudetect.c
 PORTS_SRCS-$(VPX_ARCH_MIPS) += mips.h

--- vp8/common/generic/systemdependent.c.orig	2021-10-07 01:41:19.000000000 +0800
+++ vp8/common/generic/systemdependent.c	2022-04-24 18:03:19.000000000 +0800
@@ -14,7 +14,7 @@
 #include "vpx_ports/arm.h"
 #elif VPX_ARCH_X86 || VPX_ARCH_X86_64
 #include "vpx_ports/x86.h"
-#elif VPX_ARCH_PPC
+#elif VPX_ARCH_PPC64LE
 #include "vpx_ports/ppc.h"
 #elif VPX_ARCH_MIPS
 #include "vpx_ports/mips.h"
